### ISSUE TITLE
Use `tldr` installed via Homebrew

### ISSUE THREAD

**Issue by zeitchef** [mrjones2014/tldr.nvim#1]
This looks cool, thanks! Any reason you can't hook into the `tldr` I've already got installed via Homebrew?

**Comment by mrjones2014** [comment]
It uses the global `tldr` that’s on your `$PATH` by default. But you can also configure the path to an alternate command. See [configuration](https://github.com/mrjones2014/tldr.nvim). Does that answer your question?

**Comment by zeitchef** [comment]
Right, so if I have `tldr` on my `$PATH` from Homebrew, it should just use that. Problem is it doesn't. Running all the commands in the README render nothing. 

It's only when I uninstall via Homebrew and globally install it via npm that I even see anything. However, the UI is unresponsive as well - I only ever see a spinner with either "Page not found. Updating cache..." or "Creating index..." and then finally "Error: Page not found."

<img width="1353" alt="Screen Shot 2021-10-16 at 20 22 12" src="https://user-images.githubusercontent.com/6670659/137598188-b003e15c-bdc8-4e8e-a369-5220b3051480.png">

I then tried the Python install, but it just complained that there was no `tldr` version installed in Node. Weird.

I've tried both configuration methods, neither work for me.

**Comment by mrjones2014** [comment]
Based on your screenshot it looks like there might be an issue with the result parsing. What's the output of `tldr --list` with the homebrew version?

**Comment by zeitchef** [comment]
`tldr: unrecognized option '--list'`

tldr v1.3.0

**Comment by mrjones2014** [comment]
@zeitchef ah, maybe `--list` is only an option on the Rust client. Lemme look into other ways to handle this.

**Comment by mrjones2014** [comment]
Looks like it's available on the node client. https://github.com/tldr-pages/tldr-node-client#usage

Do you know which client homebrew installs?

**Comment by mrjones2014** [comment]
It's available on the Python client as well https://github.com/tldr-pages/tldr-python-client#usage

**Comment by zeitchef** [comment]
I'm almost positive Homebrew installs the Node client, but it seems to be at least two versions behind.

**Comment by mrjones2014** [comment]
Does it work with the latest node client?

**Comment by zeitchef** [comment]
As I wrote before, I tried installing the latest Node client and I got what you see in the screenshot above.

**Comment by mrjones2014** [comment]
Ah, I found the issue. The Rust client `--list` option outputs newline separated items, while the node client outputs comma-separated items. I need to update to handle that discrepancy.

**Comment by mrjones2014** [comment]
@zeitchef please update the plugin and try again, I added support for comma-separated values in b397ce4

**Comment by zeitchef** [comment]
Cool, I can confirm this is working. The only issue is there is absolutely no syntax highlighting, but this is a known problem with the Node client.

Being a JavaScript developer, I'm switching Node versions around quite a bit for different projects. Aside from nicer aesthetics, it's really handy to have `tldr` globally available from Homebrew.

**Comment by mrjones2014** [comment]
Yeah unfortunately there's not much I can do about syntax highlighting if it's a known issue with the node client. Previewing is just using `termopen`
